Traceback (most recent call last):
  File "C:/PythonBook/ch08/Traceback.py", line 11, in <module>
    CheckSpeed(speed) #檢查速度
  File "C:/PythonBook/ch08/Traceback.py", line 5, in CheckSpeed
    raise Exception("速度太慢了!") # 拋出 Exception 型別例外
Exception: 速度太慢了!
Traceback (most recent call last):
  File "C:/PythonBook/ch08/Traceback.py", line 11, in <module>
    CheckSpeed(speed) #檢查速度
  File "C:/PythonBook/ch08/Traceback.py", line 7, in CheckSpeed
    raise Exception("已經超速了!") # 拋出 Exception 型別例外
Exception: 已經超速了!
